CHANGELOG.md

```
@@ -1,21 +1,30 @@
-  * feat: Added xsd for validating the result for AbstractMt.xml() output
-
-#### 10.2.9 - June 2025
-  * (PW-2541) Updated the narrative resolver, format field 70 (used in field 70), to allow multiple lines and using // as codeword separator (/CODEWORD1/text///CODEWORD2/text) 
-
-#### 10.2.8 - June 2025
+#### 10.3.3 - July 2025
+  * (PW-2541) Updated the narrative resolver, format field 70 (used in field 70), to allow multiple lines and using // as codeword separator (/CODEWORD1/text///CODEWORD2/text)
+  * Fix: Adding a new entry to SchemeConstantsR class
+  * Fix: Enhanced the Xml parsing logic to escape/unescape `<` and `>` characters in the XML content
   * Fix: Enhanced the `DefaultMtMetadataStrategy` to prevent NPE when the message headers are malformed
+  * Feat: Added an XSD for validating the Prowide Core proprietary XML version for MT messages generated by the `xml()` method for the `AbstractMT` and subclasses
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)